std/typecons.d

```diff
@@ -5626,17 +5626,17 @@ to deallocate the corresponding resource.
         .destroy(_refCounted._store._payload);
         static if (hasIndirections!T)
         {
-            pureGcRemoveRange(&_refCounted._store._payload);
+            (() @trusted => pureGcRemoveRange(&_refCounted._store._payload))();
         }
 
-        pureFree(_refCounted._store);
+        (() @trusted => pureFree(_refCounted._store))();
         _refCounted._store = null;
     }
 
 /**
 Assignment operators
  */
-    void opAssign(typeof(this) rhs)
+    void opAssign(typeof(this) rhs) @trusted
```

Comment:
`@trusted` for `std.algorithm.swap(_refCounted._store, rhs._refCounted._store)`. I believe this is `safe` due to the prohibition against structs holding pointers to themselves and due to the lack of any `safe` way for anyone else to have the address of either _store (meaning there is no need to account for this).